src/main/java/uk/gov/hmcts/reform/civil/notification/handlers/caseproceedsincaseman/CaseProceedsInCasemanAppSolOneEmailDTOGenerator.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.civil.notification.handlers.caseproceedsincaseman;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.notification.handlers.AppSolOneEmailDTOGenerator;
+import uk.gov.hmcts.reform.civil.notify.NotificationsProperties;
+import uk.gov.hmcts.reform.civil.service.OrganisationService;
+
+@Component
+public class CaseProceedsInCasemanAppSolOneEmailDTOGenerator extends AppSolOneEmailDTOGenerator {
+
+    private static final String REFERENCE_TEMPLATE = "case-proceeds-in-caseman-applicant-notification-%s";
+
+    protected CaseProceedsInCasemanAppSolOneEmailDTOGenerator(NotificationsProperties notificationsProperties, OrganisationService organisationService) {
+        super(notificationsProperties, organisationService);
+    }
+
+    @Override
+    protected String getEmailTemplateId(CaseData caseData) {
+        return notificationsProperties.getSolicitorCaseTakenOffline();
+    }
+
+    @Override
+    protected String getReferenceTemplate() {
+        return REFERENCE_TEMPLATE;
+    }
+
+    @Override
+    protected Boolean getShouldNotify(CaseData caseData) {
+        return caseData.isLipvLipOneVOne() ? Boolean.FALSE : Boolean.TRUE;
```

Comment:
I think this is not right. In case of lip v lr also we dont send app sol email. Just get rid of this method ., base class check should be fine